.gitignore

```diff
@@ -178,6 +178,8 @@ dist-esm/
 browser/
 types/
 dist/
+# Don't commit the sub resource integrity generated files
+**/*.integrity.json
```

Comment:
Does each extension generate a integrity.json file? Is it stored under each extension's folder?

Reply:
Yes, every extension and Sku gets it's own integrity.json file.
Generally, these are included in the browser folder (which is automatically excluded from git).
This one is included as it is also generating a integrity.json file for the snippet, which we don't need to checkin